### ISSUE TITLE
dead link in `mmdetection/configs/mask_rcnn/README.md`

### ISSUE THREAD

**Issue by kindlychung** [open-mmlab/mmdetection#3016]
**Describe the bug**
There is a dead link in `mmdetection/configs/mask_rcnn/README.md`, namely this one: [https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/mask_rcnn/mask_rcnn_x101_64x4d_fpn_1x_coco/mask_rcnn_x101_64x4d_fpn_2x_coco_20200509_224208-39d6f70c.pth](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/mask_rcnn/mask_rcnn_x101_64x4d_fpn_1x_coco/mask_rcnn_x101_64x4d_fpn_2x_coco_20200509_224208-39d6f70c.pth)

**Comment by kindlychung** [comment]
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>
mmdetection/v2.0/mask_rcnn/mask_rcnn_x101_64x4d_fpn_1x_coco/mask_rcnn_x101_64x4d_fpn_2x_coco_20200509_224208-39d6f70c.pth
</Key>
<RequestId>128E6FBE653D5FE7</RequestId>
<HostId>
UNrWreUG5YEhY4QH3yGGDm0S5S/NdwPt9tpw9mafEeJBlQL5GrmciZwCoTzvo+R/l1SrEXzejhQ=
</HostId>
</Error>
```

**Comment by yhcao6** [comment]
Thanks for reporting the error. I just fix it in #3030.
